### PR TITLE
14.0 mig website product configurator

### PR DESCRIPTION
**Pull request by PCatinean**

### CODE REVIEW

**Comment by dreispt** [comment]
Hello @PCatinean Are you planning to continue work on this?
There are bugs in the current base module. In particular all attributes are visible in all wizard steps.
I was wondering if your work will help with this.
Thanks

**Comment by PCatinean** [comment]
Hi Daniel, yes quite a few things are fixed in this PR we just have to revert all changes which undo your works and also have pre-commit checks pass. @pledrateam can you please look into this?

**Comment by dreispt** [comment]
@PCatinean the changes we did were to have pre-commits pass, so reverting them works against your goal.
It there any chances you can extract the fixes into a PR?

**Comment by PCatinean** [comment]
@dreispt we're working on it

**Comment by dreispt** [comment]
Shouldn't this also bring the full history?

File: website_product_configurator/data/config_form_templates.xml [hunk 1]
```diff
@@ -0,0 +1,994 @@
...
+                                            t-att-name="'%s%s' % (field_prefix, line.attribute_id.id)"
+                                            t-att-disabled="value.id not in available_value_ids and True or False"
+                                            t-att-title="value.name"
+                                        />
+                                    </div>
+                                    <label
+                                        class="image_config_attr_value_radio"
+                                        for="'%s%s' % (field_prefix, line.attribute_id.id)"
+                                        t-att-data-val-id="value.id"
+                                        t-attf-style="background-image:url(/web/image/product.attribute.value/{{value.id}}/image_medium);"
```
Comment (CasVissers-360ERP) [inline website_product_configurator/data/config_form_templates.xml]: Some initial testing, for the images the work you need:
t-attf-style="background-image:url(/web/image?model=product.attribute.value&amp;id={{value.id}}&amp;field=image);

File: website_product_configurator/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+        website=True,
+    )
+    def cfg_session(self, cfg_session_id, **post):
+        """Render product page of product_id"""
+        if (
+            not cfg_session_id.exists()
+            or cfg_session_id.user_id != request.env.user
+            or cfg_session_id.state != "done"
+        ):
+            return request.render("website_sale.404")
```
Comment (CasVissers-360ERP) [inline website_product_configurator/controllers/main.py]: website.page_404

**Comment by PCatinean** [comment]
@pledrateam codecoverage is failing

**Comment by PCatinean** [comment]
Test have passed with coverage increase, can you please review @dreispt @CasVissers-360ERP ?

**Review by dreispt — APPROVED** [review]

**Pull request merged**